state/state.go

```diff
@@ -8,16 +8,14 @@ import (
 	"github.com/Vivena/babelweb2/parser"
 )
 
-type Transition parser.Transition
```

Comment:
I just remove it because there is not benefit of declare it again.

You do not extents this datatype by something like: 
```golang
func (set *Transition) String() string {
```

For me it is okay, like it is used (across multiple packages).

Reply:
> For me it is okay, like it is used (across multiple packages).

For me not.